client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -15,22 +14,19 @@ import FollowButton from 'reader/follow-button';
 import { getStreamUrl } from 'reader/route';
 import EmailSettings from './email-settings';
 
-const stripUrl = url => url.replace( 'https://', '' ).replace( 'http://', '' ).replace( '/', '' );
-
 function ReaderSubscriptionListItem( {
 	url,
 	feedId,
 	feed,
 	siteId,
 	site,
 	className = '',
-	followSource,
-	lastUpdated,
 	translate,
+	followSource,
 } ) {
-	const siteTitle = feed && feed.name;
+	const siteTitle = ( feed && feed.name ) || ( site && site.title );
```

Comment:
Seems like there are a few useful utilities that we could make here because they get reused in a couple of places:
```
getTitle( { feed, site, post } ) --> title
getAuthorName( { feed, site, post} ) --> author name
```

for now I'll jut flip it.  I'll create a followup pr that makes testable/reusable bits for post & streams & manage